src/platform/forms-system/src/js/fields/ObjectField.jsx

```diff
@@ -290,11 +283,84 @@ class ObjectField extends React.Component {
       </div>
     );
 
+    const accessibleFieldContent = (
+      <>
+        {hasTitleOrDescription && (
+          <>
```

Comment:
This whole thing is a copy of `fieldContent`, but the only difference is these two fragments being here instead of being `<div>`s with className attributes.

Reply:
The `fieldContent` I'm talking about:

https://github.com/department-of-veterans-affairs/vets-website/blob/e05f44309e62b09996f95977befd841e0c667220/src/platform/forms-system/src/js/fields/ObjectField.jsx#L255-L315